invenio_oaiserver/utils.py

```diff
@@ -79,3 +107,86 @@ def datetime_to_datestamp(dt, day_granularity=False):
     if day_granularity:
         result = result[:-10]
     return result
+
+
+def eprints_description(content=None, metadataPolicy=None, dataPolicy=None,
+                        submissionPolicy=None):
+    """Generate the eprints element for the identify response.
+
+    The eprints container is used by the e-print community to describe
+    the content and policies of repositories.
+    For the full specification and schema definition visit:
+    http://www.openarchives.org/OAI/2.0/guidelines-eprints.htm
+    """
+    eprints = Element(etree.QName(NS_EPRINTS[None], 'eprints'),
+                      nsmap=NS_EPRINTS)
+    eprints.set(etree.QName(ns['xsi'], 'schemaLocation'),
+                '{0} {1}'.format(EPRINTS_SCHEMA_LOCATION,
+                                 EPRINTS_SCHEMA_LOCATION_XSD))
+
+    if content:
+        contentElement = etree.Element('content')
+        for key, value in content.items():
+            contentElement.append(E(key, value))
+        eprints.append(contentElement)
+
+    if not metadataPolicy:
+        raise OAIMissingRequiredElementError('metadataPolicy')
+    metadataPolicyElement = etree.Element('metadataPolicy')
+    for key, value in metadataPolicy.items():
+        metadataPolicyElement.append(E(key, value))
+        eprints.append(metadataPolicyElement)
+
+    if not metadataPolicy:
```

Comment:
``if not dataPolicy``? take a look at the comment above though, this could be positional instead.